dashboard/app/models/teacher_feedback.rb

```diff
@@ -63,12 +65,17 @@ def self.get_latest_feedback_given(student_id, level_id, teacher_id, script_id)
   end
 
   # returns the latest feedback from each teacher for the student on the level
-  def self.get_latest_feedbacks_received(student_id, level_id, script_id)
-    where(
+  def self.get_latest_feedbacks_received(student_id, level_id, script_id, review_state = nil)
```

Comment:
If a level has two feedback with `review_state` equal to `keepWorking` and `completed` (in that order), which feedback is returned?  Put another way, is the `review_state` filter applied before or after the `latest` filter?

Reply:
Oh! Excellent point, this is not going to work for my use case, because you're right. I only want to get the feedback with keepWorking if it's also the latest feedback. Thank you for pointing this out! I will fix